disconnect-when-backgrounded.js

```diff
@@ -1,18 +1,30 @@
 var disconnectTimer = null;
 
+// 10 seconds by default
+var disconnectTime = (Meteor.settings && Meteor.settings.public && Meteor.settings.public.disconnectTime || 10) * 1000;
```

Comment:
I would also call this `disconnectTimeSec` since it's a bit unconventional to use seconds instead of milliseconds for time durations in javascript.